plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -585,21 +586,41 @@ global proc mayaUsdMenu_layerEditorContextMenu(string $panelName) {
     $enabled = 1;
     menuItem -label $label -enable $enabled -c $cmd;
 
+    if($isIncoming) {
+        $incoming = `listConnections -s true -d false -c true $proxyShape`;
+        $incomingNode = "";
+        for($i=0;$i<size($incoming);$i+=2) {
+            $j = $i+1;
+            if($incoming[$i] == $proxyShape+".inStageData") {
+                $incomingNode = $incoming[$j];
+            }
+            else if($incomingNode == "" && $incoming[$i] ==  $proxyShape+".stageCacheId") {
+                $incomingNode = $incoming[$j];
+            }
```

Comment:
Can you add a comment as to what this code is doing?

Reply:
General question: I saw that you resolved this conversation, but I don't see any new comment. I would prefer if the conversations were left open and resolved by the person who created them.

Reply:
The resolution means I addressed in my code, it would have been marked automatically anyway because the code is outdated. For me its a way to keep track of what I resolved or not

Reply:
I haven't pushed my code since I didnt want to push many small changes, I wanted to address all the comments in one change.